.gitignore

```diff
@@ -73,3 +73,7 @@ fastlane/screenshots
 Screenshots/FailureDiffs/*
 
 Frameworks/OpenTok/*
+
+# Ignore Jetbrains local project cash directory
```

Comment:
You mean "cache"? What app generates this file? If it's not iOS workflow-specific wanna add it to your global `~/.gitignore` file instead?